test/lit.cfg

```diff
@@ -53,7 +53,9 @@ def darwin_sdk_build_version_split(version):
 
 
 def darwin_sdk_build_version_cmp(lhs, rhs):
-    return cmp(
+    def _cmp(a, b):
```

Comment:
Maybe even adding a [link](https://docs.python.org/3.0/whatsnew/3.0.html#ordering-comparisons). Because the choice of expression is also interesting =) It certainly raised a question in my head: 'why not simply `a - b`?".